content/03.categorize.md

```diff
@@ -690,7 +687,7 @@ Despite the difficulties and uncertainties, machine learning practitioners (and
 particularly those who use deep neural networks, which are challenging to
 interpret) must remain cognizant of these dangers and make every effort to
 prevent harm from discriminatory predictions. To reach their potential in this
-domain, deep learning methods will need to be interpretable. Researchers need to
+domain, deep learning methods will need to be interpretable (more in the discussion section). Researchers need to
```

Comment:
Change `(more in the discussion section)` to `(see Discussion)` to be consistent with other sections.

Reply:
Agree!